WalletWasabi.Fluent/Controls/TagsBox.axaml.cs

```diff
@@ -272,8 +272,7 @@ private void CheckIsCurrentTextValid()
 			IsCurrentTextValid = !string.IsNullOrEmpty(correctedInput);
 			return;
 		}
-
-		throw new InvalidOperationException($"Invalid configuration! {nameof(Suggestions)} are not set!");
+		IsCurrentTextValid = false;
```

Comment:
@Onyx2406 Add an empty line before this one, that is better for readability of the code, and that is how it was before.

```suggestion

		IsCurrentTextValid = false;
```

Reply:
Done